s32/mcux/devices/S32K146/S32K146_features.h

```diff
@@ -301,4 +301,8 @@
 /* @brief Has Oscillator Enable(bitfield CR[OSCE]). */
 #define FSL_FEATURE_RTC_HAS_NO_CR_OSCE (1)
 
+/* ADC module features */
+
+#define FSL_FEATURE_ADC12_CONVERSION_CONTROL_COUNT (24)
```

Comment:
in the top of this file, add `#define FSL_FEATURE_SOC_ADC12_COUNT` with the number of instances availables on this device

Reply:
ok